container/test-app/wc/clientAPI.html

```diff
@@ -10,11 +10,16 @@ <h3>
       component based microfrontend
     </h3>
     <button id="luigi-update-context" type="button">Update context</button>
+    <button id="init-button" style="margin: 25px 0 15px">
+      container.init() to initialize defered container
+    </button>
+    <button id="sendCustomMessageBtn" style="margin: 25px 0 15px">sendCustomMessage</button>
 
     <div style="height: fit-content;">
       <div style="border: 1px solid blue">
         <!-- Luigi Container to test general functionalities, params etc-->
         <luigi-container
+          id="test-id"
```

Comment:
you already have **data-test-id**, you can fetch the container using that on Cypress. If you do need it, give it a more specific id names so its distinguishable from other containers in this file.

Reply:
Ok I'll remove it, I'll use the data-test-id